index.html

```diff
@@ -16,46 +16,62 @@
         <a href="#" class="logo"> Welcome </a>
 
         <nav>
-            <div>
+
+            <ul class="nav-list">
+                <a href="#portfolio">
+                    <li>Portfolio</li>
+                </a>
+                <a href="#about">
+                    <li>About</li>
+                </a>
+                <a href="#contact">
+                    <li>Contact</li>
+                </a>
+            </ul>
+
+            <div class="nav-menu">
                 <img src="./images/menu.svg" alt="menu-icon" class="menu">
             </div>
         </nav>
 
     </header>
 
```

Comment:
- Your styles are not reflecting on GitHub, kindly update the link to 
`<link rel="stylesheet" href="./styles/main.css">` to fix this issue.